recipe/activate-gcc.sh

```diff
@@ -89,13 +89,15 @@ if [ "${CONDA_BUILD:-0}" = "1" ]; then
   CPPFLAGS_USED="@CPPFLAGS@ -I${PREFIX}/include"
   DEBUG_CPPFLAGS_USED="@DEBUG_CPPFLAGS@ -I${PREFIX}/include"
   CMAKE_PREFIX_PATH_USED="${CMAKE_PREFIX_PATH}:${PREFIX}:${BUILD_PREFIX}/${HOST}/sysroot/usr"
+  CONDA_BUILD_SYSROOT_USED="${BUILD_PREFIX}/${HOST}/sysroot"
```

Comment:
```suggestion
  CONDA_BUILD_SYSROOT_USED="${BUILD_PREFIX}/@CHOST@/sysroot"
```

Reply:
@isuruf , I'll change it, but is the `BUILD_PREFIX` part correct?